code/datums/martial.dm

```diff
@@ -436,6 +436,26 @@
 		name = "empty scroll"
 		icon_state = "blankscroll"
 
+/obj/item/weapon/CQC_scroll
```

Comment:
This is just mundane CQC, so maybe make this less magical. Maybe a training manual or something?